File: 프로그래머스/1/12926. 시저 암호/시저 암호.js
function solution(s, n) {
    
    var answer = '';
    
    for(let i=0; i<s.length; i++){
        let char = s.charAt(i);
        let ascii = char.charCodeAt(0);
        if(ascii >= 65 && ascii <= 90){
            ascii -= 65;
            ascii = (ascii+n)%26;
            ascii += 65;
        }else if(ascii >= 97 && ascii <= 122){
            ascii -= 97;
            ascii = (ascii+n)%26;
            ascii += 97;
        }else{
            answer = answer + ' ';
            continue;
        }
        answer = answer + String.fromCharCode(ascii);
    }
    
    return answer;
}

File: 프로그래머스/1/12926. 시저 암호/시저 암호.java
class Solution {
    public String solution(String s, int n) {
        
        String answer = "";
        
        for(int i=0; i<s.length(); i++){
            char alpha = s.charAt(i);
            int ascii = (int)alpha;
            if(ascii >= 65 && ascii <= 90){
                ascii -= 65;
                ascii = (ascii + n)%26;
                ascii += 65;
            }else if(ascii >= 97 && ascii <=122){
                ascii -= 97;
                ascii = (ascii + n)%26;
                ascii += 97;
            }else{
                answer = answer + " ";
                continue;
            }
            answer = answer + (char)ascii;
        }
        
        return answer;
    }
}